data/tribes/initialization/empire/units.lua

```diff
@@ -1210,13 +1253,21 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_well",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Well
+            lore = pgettext("empire_building", "‘Vas redit ad fontem donec destruat.’ (The jar returns to the well until it breaks.)"),
```

Comment:
@klaus-halfmann Wow, is this really from latin? I couldn't find it anywhere.

Reply:
I retranslated it from german, have fun.

Reply:
If it's not Latin original, then we may consider something else. I had in mind "Gutta cavat lapidem…" for example.

Also google translate (for what it's worth) doesn't want to translate "destruat" to "it is destroyed", but I can't judge that myself.

Reply:
*Destruere* means *destroy*, so I think *destruat* means "until the jar destroys [something else, possibly the well]." We need the participle perfect passive here: *destructum*.